app/src/main/java/de/westnordost/streetcomplete/data/maptiles/MapTilesDownloadCacheConfig.kt

```diff
@@ -18,6 +18,13 @@ class MapTilesDownloadCacheConfig(context: Context) {
         .maxStale(DELETE_OLD_DATA_AFTER.toInt(), TimeUnit.MILLISECONDS)
         .build()
 
+    /* use separate cache control for tangram with large maxStale value to always show available
+    *  map tiles when panning the map, even without (or with bad) internet connection */
+    val tangramCacheControl = CacheControl.Builder()
+        .maxAge(12, TimeUnit.HOURS)
+        .maxStale(DELETE_OLD_DATA_AFTER.toInt(), TimeUnit.SECONDS)
```

Comment:
Wha is max stale if it is not set? Can it not simply not be set? If it must be set, maybe set to one year or something, reusing the DELETE_OLD_DATA_AFTER constant but interpreting the delay in seconds is confusing.

Reply:
Ok, I'll change it to some really long time explicitly.

> Can it not simply not be set?

Then stale cache will not be used, see https://github.com/square/okhttp/blob/master/okhttp/src/jvmMain/kotlin/okhttp3/CacheControl.kt#L172-L183